app/controllers/standup_meeting_groups/joins_controller.rb

```diff
@@ -7,12 +8,8 @@ def create
 
     authorize @standup_meeting_group_user, policy_class: StandupMeetingGroup::JoinPolicy
 
-    @standup_meeting_group_user.save!
-
-    respond_to do |format|
-      format.turbo_stream do
-        flash.now[:notice] = "You have joined #{@standup_meeting_group.name}!"
-      end
+    if @standup_meeting_group_user.save
+      flash.now[:notice] = "You have joined #{@standup_meeting_group.name}!"
     end
```

Comment:
Because there is no redirection after the save, it seems to be the case that the user cannot attempt to join the same standup meeting group via the application UI. If there was a redirection, I’d assume that the possibility of the user refreshing the browser during the request needs to be handled. For this reason, the save failure is not explicitly handled for now.